functions/system_ip_maping.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+
+def get_system_to_ip_map(system_names,system_ips):
+	""" Get lists of system names and system IPs
+		then prints the list of tuple which has system name 
+		and corresponding system IP.
+		
+		Args:
+			system_names:  It is list of system names.
+			system_ips:  It is list of system IPs
+
+        Returns:
+            Returns list of tuple having system name and 
+			corresponding system IP.
+    """
+
+	system_ip_map = []
+	if len(system_names) != len(system_ips) :
```

Comment:
More pythonic way of finding minimum of two min(len(system_names), len(system_ips) )